src/main/java/com/ognwan/controller/AuthenticationController.java

```diff
@@ -62,4 +61,15 @@ public ResponseEntity<?> login(@RequestParam String email, String password) {
 		}
 		return ResponseEntity.badRequest().body("incorrect username or password, please try again");
```

Comment:
All looks good, :)

Just a style suggestion: You could capitalize on the first work in the requests sent to the body like in this line case

`.body("incorrect...` to `.body("Incorrect...`